conda/recipes/strings_udf/meta.yaml

```diff
@@ -50,12 +62,12 @@ requirements:
     - cudf ={{ version }}
     - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
     - cachetools
-    - ptxcompiler >=0.7.0  # [linux64]  # CUDA enhanced compatibility. See https://github.com/rapidsai/ptxcompiler
-test:                                   # [linux64]
-  requires:                             # [linux64]
-    - cudatoolkit {{ cuda_version }}.*  # [linux64]
-  imports:                              # [linux64]
-    - strings_udf                       # [linux64]
+    - ptxcompiler >=0.7.0  # CUDA enhanced compatibility. See https://github.com/rapidsai/ptxcompiler
+test:
```

Comment:
This is failing on ARM + gpuCI Jenkins. We probably can't remove the `# [linux64]` until Jenkins is disabled. This appears to work fine on GitHub Actions for whatever reason.